### DISCUSSION TITLE
Can’t seem to play AVC1 files with HD300 - anyone else?

### DISCUSSION THREAD

**Discussion by Sagetvmaster** [OpenSageTV/Community#81] [SageTV Extenders (HD300,HD200)]
I know H265 won’t work, but these are the first AVC1 files somebody sent me.  Anyone else have a problem, or know a work around?

**Comment by mickgeller** [comment]
As I wrote this I checked and confirmed that I can watch H.265 on my HD200 and HD300 extenders. I recommend loading two plugins. They are related, as they use the same, updated, version of FFMPEG. One is "SageTV Transcoder (FFmpeg)." The other is "Media Format Parser Plugin." As long as FFMPEG supports AVC1 you should be OK.

**Reply by Sagetvmaster** [reply]
Thanks, I’ll try this :)

**Reply by wayner9** [reply]
Let us know if it worked.

**Reply by Sagetvmaster** [reply]
Turns out I have both those plugins installed, so it must be something else - maybe the audio codec that's being used.  I reformated them in Handbrake and they work fine now.  I'll dig deeper.  Thanks

**Reply by trk2** [reply]
There is a setting in the server property value that disables transcoding which could be causing your issue.  Unfortunately I don't remember the name.

**Comment by LoisGNS** [comment]
Which encoders output AVC1 files? I'm not familiar with them.

**Comment by emveepee** [comment]
Is it possibly you are confusing AV1 https://www.xda-developers.com/av1/ with AVC.  AV1 is really only supported in h/w on newer devices.
